packages/www/pages/jobs.tsx

```diff
@@ -0,0 +1,65 @@
+import { useRouter } from "next/router";
+import Fade from "react-reveal/Fade";
+import { Element } from "react-scroll";
+import { Box } from "@livepeer.com/design-system";
+import Hero from "components/Marketing/Hero";
+import Layout from "layouts/main";
+import { getJobs } from "hooks";
```

Comment:
`getJobs` isn't being used as a hook. Let's define somewhere else. Also, hooks should be prefixed with `use`.